### ISSUE TITLE
Experimental build is really out of date - builds are not updating.

### ISSUE THREAD

**Issue by tomdee** [docker/docker#14291]
Using either curl experimental.docker.com|sh or grabbing the binary directly (https://experimental.docker.com/builds/Linux/x86_64/docker-latest)
I can only get a build from 8 days ago - at this revision - https://github.com/docker/docker/commit/f39b9a0b0f77c528a6fb8bf14be5cdea11ce338d

```
Client version: 1.8.0-dev
Client API version: 1.20
Go version (client): go1.4.2
Git commit (client): f39b9a0
OS/Arch (client): linux/amd64
Experimental (client): true
Server version: 1.8.0-dev
Server API version: 1.20
Go version (server): go1.4.2
Git commit (server): f39b9a0
OS/Arch (server): linux/amd64
Experimental (server): true

If you would like to use Docker as a non-root user, you should now consider
adding your user to the "docker" group with something like:

  sudo usermod -aG docker your-user

Remember that you will have to log out and back in for this to take effect!

gulfstream@gulfstream:~/calico-docker$ wget https://experimental.docker.com/builds/Linux/x86_64/docker-latest
--2015-06-29 16:13:48--  https://experimental.docker.com/builds/Linux/x86_64/docker-latest
Resolving experimental.docker.com (experimental.docker.com)... 54.230.117.147, 54.230.117.241, 54.239.132.111, ...
Connecting to experimental.docker.com (experimental.docker.com)|54.230.117.147|:443... connected.
HTTP request sent, awaiting response... 200 OK
Length: 19380581 (18M) [binary/octet-stream]
Saving to: ‘docker-latest’

docker-latest       100%[=====================>]  18.48M  1.27MB/s   in 19s    

2015-06-29 16:14:07 (987 KB/s) - ‘docker-latest’ saved [19380581/19380581]

gulfstream@gulfstream:~/calico-docker$ chmod +x docker-latest 
gulfstream@gulfstream:~/calico-docker$ ./docker-latest -v
Docker version 1.8.0-dev, build f39b9a0, experimental
gulfstream@gulfstream:~/calico-docker$ ./docker-latest version
Client version: 1.8.0-dev
Client API version: 1.20
Go version (client): go1.4.2
Git commit (client): f39b9a0
OS/Arch (client): linux/amd64
Experimental (client): true
Server version: 1.8.0-dev
Server API version: 1.20
Go version (server): go1.4.2
Git commit (server): f39b9a0
OS/Arch (server): linux/amd64
Experimental (server): true
```

**Comment by tomdee** [comment]
ping @jfrazelle Are you still in charge of builds? Am I wrong to think that these binaries should be updated more regularly?

**Comment by tomdee** [comment]
For anyone following this issue, I discussed with @icecrime on IRC and got this response

icecrime> tomdee: it's acknowleged, I forgot about that but we had the problem last week already, when no commits happen in a timeframe, we try to push the same hash twice to the apt repo, and we get rejected
11:39 AM <icecrime> tomdee: the build job is off right now, should be fix in the next few days

**Comment by icecrime** [comment]
Closed by #14064.
